bottomless/src/replicator.rs

```diff
@@ -155,12 +155,19 @@ impl Options {
     pub fn from_env() -> Result<Self> {
         fn env_var(key: &str) -> Result<String> {
             match std::env::var(key) {
-                Ok(res) => Ok(res),
+                Ok(res) => {
+                    let res = res.trim().to_string();
+                    if res.is_empty() {
+                        bail!("{} environment variable is empty", key)
+                    } else {
+                        Ok(res)
+                    }
+                }
```

Comment:
If we don't trim, then it lets us slip in values containing just spaces. I don't think we ever want env variables to have spaces, trimming seems like a must.